src/SpectatorView.Unity/Assets/PhotoCapture/Scripts/HoloLensCamera.cs

```diff
@@ -5,6 +5,8 @@
 // identity 'System.Numerics.Vectors, Version=4.1.1.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' of 'System.Numerics.Vectors', you may need to supply runtime policy
 #pragma warning disable 1701
 
+#pragma warning disable CS0618 // 'WorldManager' is obsolete: 'Support for built-in VR will be removed in Unity 2020.1.
```

Comment:
Could you wrap the exact line (if not many) with this "#pragma warning disable" and "#pragma warning restore"

Reply:
It's generally nicer to do what @andreiborodin mentions because it keeps the warning ignores specific to the locations that need them (and thus makes each new warning explicitly annoying because don't want people adding more).

I think where this is fine though is if the entire file is also deprecated/non-sense without that dependency (i.e. if HoloLensCamera will be deleted in 2020) then it makes sense to have at the top level.

Reply:
I'm going to leave this how it is for now, these files will all have various issues in the future.